src/main/java/com/exadel/sober/controllers/PromiseController.java

```diff
@@ -0,0 +1,29 @@
+package com.exadel.sober.controllers;
+
+import com.exadel.sober.models.Promise;
+import com.exadel.sober.repositories.PromiseRepository;
+import com.exadel.sober.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin
+@RequestMapping(path="/promise")
+public class PromiseController {
+
+    @Autowired
+    private PromiseRepository promiseRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @PostMapping(path="/deletePromise")
```

Comment:
If you are trying to delete something - use specific http method for that DELETE